docs/user-guide/configure-certificates.md

```diff
@@ -1,30 +1,77 @@
-# Configuring Zowe certificates 
+# Zowe certificate configuration overview
 
-As a system administrator, review this article to learn about the key concepts of Zowe certificates.
+As a system programmer, review this article to learn about the key concepts of Zowe certificates, and options for certificate configuration. 
 
-Zowe uses a certificate to encrypt data for communication across secure sockets. The certificate used is specified in the `zowe.certificates` section of the `zowe.yaml`, and can either be a `PKCS12` certificate held in a USS keystore `.p12` file, or else a `JCERACKS` certificate held in a keyring.  
+Zowe uses digital certificates to verify the identity and subsequently establish an encrypted network connection between applications using the Secure Sockets Layer/Transport Layer Security (SSL/TLS) protocol. Certificates and their associated private keys need to be stored either in the SAF key ring or in the `PKCS12` java keystore.
 
-Zowe provides the ability to generate a certificate using the `zwe init certificate` command, and it can also be configured to use an existing certificate that might be provided by the security team in a z/OS customer shop.
+Zowe provides the ability to generate a certificate using the `zwe init certificate` command. Zowe can also be configured to use an existing certificate provided by the user's security team in a z/OS customer shop.
  
 ## Trust store
```

Comment:
This appears out of nowhere - users will ask what's a truststore?
Point is, might want to reorganize this page for progressive disclosure.

I imagine the page would benefit organized as:


* As a collection of software that does networking, Zowe uses certificates for SSL/TLS & HTTPS.
* Certificates and their verification are managed within a keystore (certificate storage) and a truststore (verification storage)
* Zowe supports keystores and truststores that are either z/OS keyrings (when on z/OS) or PKCS12 files
* Zowe provides utilities for helping you generate the keystore & truststore, or re-use ones you already have. (zwe init certificate / JCLs)
* Zowe records the final configuration in the yaml.